Resources/Prototypes/Entities/Clothing/Uniforms/jumpskirts.yml

```diff
@@ -126,9 +126,9 @@
   description: It's a turtleneck worn by those with the experience to be Chief Medical Officer. It provides minor biological protection.
   components:
   - type: Sprite
-    sprite: Clothing/Uniforms/Jumpskirt/cmo_turtle.rsi
+    sprite: _CD/Clothing/Uniforms/Jumpskirt/cmoturtleskirt.rsi # CD
   - type: Clothing
-    sprite: Clothing/Uniforms/Jumpskirt/cmo_turtle.rsi
+    sprite: _CD/Clothing/Uniforms/Jumpskirt/cmoturtleskirt.rsi # CD
```

Comment:
Any reason why you are replacing these instead of creating a new item?

Reply:
its easier then replacing every reference to the original manually

Reply:
Yeah this is fine, it's extremely similar to the original design just done a little differently with different colors.